webapps/waterfall-chart/backend.py

```diff
@@ -0,0 +1,65 @@
+import dataiku
+from flask import request
+import pandas as pd
+import numpy as np
+import json
+import traceback
+import logging
+logger = logging.getLogger(__name__)
+
+
+def build_complete_df(df, category_column, value_column, max_displayed_values, group_others):
+    try:
```

Comment:
Some comments would be welcomed:
@see https://www.python.org/dev/peps/pep-0257/ and the plugin guidelines .

In short, explain what the method do, and what is not trivial to guess by reading the code.